sdks/python/apache_beam/io/gcp/pubsub_test.py

```diff
@@ -172,6 +205,118 @@ def test_display_data(self):
     hc.assert_that(dd.items, hc.contains_inanyorder(*expected_items))
 
 
+class TestPubSubReadEvaluator(object):
```

Comment:
<!--new_thread; commit:6bfe8d93b2cb863f16655addcbb98b35d64ce365; resolved:0-->
Do we want to inherit from `TransformEvaluator`?  (though it doesn't matter functionality-wise)

Reply:
Not inheriting from TransformEvaluator means that we can be sure that _PubSubReadEvaluator methods are always being used, so I'd rather keep it as is.